v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-single-model/pipeline_helper.py

```diff
@@ -135,6 +137,10 @@ def get_backtest_pipeline(
     if model_uid is not None:
         prs_args.append("--model-uid")
         prs_args.append(model_uid)
+    if forecast_quantiles:
+        prs_args.append("--forecast_quantiles")
+        for val in range(len(forecast_quantiles)):
+            prs_args.append(forecast_quantiles[val])
```

Comment:
Let's simplify this:

prs_args.extend(forecast_quantiles)

Reply:
Done. Thanks